terraform/monitoring/variables.tf

```diff
@@ -3,12 +3,18 @@ variable paas_user { default = null }
 variable paas_password { default = null }
 
 locals {
-  service_name             = "teaching-vacancies"
-  monitoring_instance_name = local.service_name
-  paas_api_url             = "https://api.london.cloud.service.gov.uk"
-  monitoring_org_name      = "dfe-teacher-services"
-  space_name               = "${local.service_name}-monitoring"
-  monitoring_space_name    = "${local.service_name}-monitoring"
-  aws_region               = "eu-west-2"
-  secrets                  = yamldecode(data.aws_ssm_parameter.monitoring_secrets.value)
+  service_name               = "teaching-vacancies"
+  monitoring_instance_name   = local.service_name
+  paas_api_url               = "https://api.london.cloud.service.gov.uk"
+  monitoring_org_name        = "dfe-teacher-services"
+  space_name                 = "${local.service_name}-monitoring"
+  monitoring_space_name      = "${local.service_name}-monitoring"
+  aws_region                 = "eu-west-2"
+  secrets                    = yamldecode(data.aws_ssm_parameter.monitoring_secrets.value)
+  alertmanager_slack_url     = local.secrets["alertmanager_slack_url"]
+  alertmanager_slack_channel = "twd_tv_dev"
+  alertmanager_config = templatefile("${path.module}/config/alertmanager.yml.tmpl", {
```

Comment:
This can be removed

Reply:
I think lines 14 and 15 should still stand, as we need to send the teaching-vacancies specific values, but line 16 can be removed